src/Microsoft.Health.Fhir.Core/Features/Conformance/Serialization/CodingJsonConverter.cs

```diff
@@ -0,0 +1,66 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using EnsureThat;
+using Hl7.Fhir.Model;
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.Fhir.Core.Features.Conformance.Serialization
+{
+    /// <summary>
+    /// Converts <see cref="Coding"/> into a simple serializable model for <see cref="CapabilityStatementBuilder"/>
+    /// </summary>
+    internal class CodingJsonConverter : JsonConverter
```

Comment:
>CodingJsonConverter [](start = 19, length = 19)

just curious, what it show up in json if you don't use this converter?

Reply:
The `CodingElement` fields are serialized (https://github.com/FirelyTeam/firely-net-common/blob/develop/src/Hl7.Fhir.Support.Poco/Model/Generated/Coding.cs)

Reply:
I feel silly. I've been calling metadata, but it's not been writing at all (probably because only coding in metadata is messaging.endpoint.protocol and we don't output it).

What spark it creation if never hit it? Some experemental branch?

Reply:
Do you have auth turned on?

Reply:
🤦